exporter/kafkaexporter/config.go

```diff
@@ -37,6 +37,9 @@ type Config struct {
 
 	// Authentication defines used authentication mechanism.
 	Authentication Authentication `mapstructure:"auth"`
+
+	// ProxyURL is the url for the proxy to be used with sarama
+	ProxyURL string `mapstructure:"proxy_url"`
```

Comment:
@padraicbc do we need to have support directly in the config? How is this different to `HTTP_PROXY` env var supported by golang?

Reply:
Sarama doesn't pick up envs and you won't be using a http proxy.  We had squid running and tried numerous ways to make it play nice but you need to tunnel over a TCP proxy as we are talking Kafka, not HTTP.

Reply:
I had used envs but it wasn't the preferred way. All this change does is alllow pointing to a TCP proxy you have registered. It is up to the end  user to create  and register  it.